ephyviewer/epochencoder.py

```diff
@@ -93,10 +95,11 @@ def make_params(self):
         self.params.param('xsize').setLimits((0, np.inf))
         
         
-        keys = 'azertyuiop'
+        keys = '1234567890'
```

Comment:
OK for this.
'azertyuiop' is the first line of a french keyboard.
Now ephyviewer is becoming an international tools!!!